tests/unit/services/storefront-test.js

```diff
@@ -1,36 +0,0 @@
-import { moduleFor, test } from 'ember-qunit';
-
-moduleFor('service:storefront', 'Unit | Service | storefront', {
-  // Specify the other units that are required for this test.
-  //needs: ['service:store']
-});
-
-test('_key will generate a blank key for an empty query', function(assert) {
-  let query = {};
-  let key = "model:";
-  assert.equal(this.subject()._key('model', query), key);
-});
-
-test('_key will generate a key for a query', function(assert) {
-  let query = { testing: 123 };
-  let key = "model:testing=123";
-  assert.equal(this.subject()._key('model', query), key);
-});
-
-test('_key will generate a key for a nested object', function(assert) {
-  let query = { filter: { testing: 123 } };
-  let key = "model:filter[testing]=123";
-  assert.equal(this.subject()._key('model', query), key);
-});
-
-test('_key will generate a key for a nested array', function(assert) {
-  let query = { testing: [1, 2, 3] };
-  let key = "model:testing[]=1&testing[]=2&testing[]=3";
-  assert.equal(this.subject()._key('model', query), key);
-});
-
-test('_key will generate a key for boolean values', function(assert) {
-  let query = { true: true, false: false };
-  let key = "model:true=true&false=false";
-  assert.equal(this.subject()._key('model', query), key);
-});
```

Comment:
I think these should be tested somewhere.

Reply:
I felt like these tests were too coupled to the implementation so I tried to cover the public API of these tests with this one: https://github.com/EmberMap/ember-data-storefront/blob/add-load-record/tests/integration/-private/cache-test.js

Reply:
oh perfect, exactly what i meant by 'somewhere'